[OSDOCS#17176]: Make changes as per the errors and warnings for CQA-1 (#114179)
openshift/openshift-docs: modules/security-understanding-openshift.adoc

[id="security-understanding-openshift_{context}"]
= What is {product-title}?

[role="_abstract"]

> {product-title} automates the deployment, operation, and management of containerized applications by using Kubernetes as its orchestration engine, enhanced with Red Hat security, support, and enterprise features.

Reviewer: ```suggestion
You can use {product-title} to automate the deployment, operation, and management of containerized applications.  {product-title} uses Kubernetes as its orchestration engine, enhanced with Red Hat security, support, and enterprise features.
```
